docs/api/java/index.md

```diff
@@ -0,0 +1,12 @@
+# MXNet - Java Inference API
+
+MXNet supports Java for performing inference on a trained model. The MXNet Java Inference API is an extension of the [Scala Infer API](../../api/scala/infer.html) which provides model loading and inference functionality.
+
+The goal of the MXNet Java package is to provide an efficient and easy to use inference API.
+The MXNet Java package makes it easy to quickly deploy an existing model into a production level Java ecosystem.
+
+## Tutorials
+See the [tutorial page](../../tutorials/index.html) for detailed tutorials and examples using the Java Inference API.
```

Comment:
Add these to the tutorials page, with an anchor, and link to that instead of just the tutorials root:

tutorials/java/mxnet_java_on_intellij.md
tutorials/java/ssd_inference.md

Another route that will fix toc errors, is to add an index.md page in the tutorials/java/ folder. This will get rid of this error that I'm seeing now:

```
/home/ubuntu/incubator-mxnet/docs/tutorials/java/mxnet_java_on_intellij.md:: WARNING: document isn't included in any toctree
/home/ubuntu/incubator-mxnet/docs/tutorials/java/ssd_inference.md:: WARNING: document isn't included in any toctree
```
Then you can link to that page instead.